Wiki/zh-Tw.js

```diff
@@ -80,7 +81,8 @@ exports.GENERAL = {
   MESSAGE: "訊息",
   USER: "使用者",
   SERVER: "伺服器",
-  NAME: "名稱"
+  NAME: "名稱",
+  URL: "URL"
```

Comment:
A translation from Traditional Chinese is missing here.

Reply:
I think it can be URL as well as it in Chinese

Reply:
@kami-sqmf Ok 👍